package.json

```diff
@@ -26,18 +26,19 @@
     "extends browserslist-config-terra"
```

Comment:
should this be 
```suggestion
    "extends ./packages/browserslist-config-terra"
```

Reply:
This actually doesn't work.  You can't configure browserslist that way.  I had to make it this way to reference a local file.  The local file then imports ./packages/browserslist-config-terra.